models/user/user_system.go

```diff
@@ -68,3 +68,19 @@ func NewActionsUser() *User {
 func (u *User) IsActions() bool {
 	return u != nil && u.ID == ActionsUserID
 }
+
+func NewCLIUser() *User {
```

Comment:
Audit logging is not only useful in the case of bad intentions, but to reconstruct why something has changed the way it did.

Reply:
I don't think so. AuditLog is not a log. If it log everything, what's the difference between it and action behaviors?

Reply:
> AuditLog is not a log

Why not?

I'd say that is exactly its purpose as @johanvdw wrote: to reconstruct why something has changed the way it did.
And omitting this crucial info that the CLI was used does not help anyone

Reply:
> Audit logging is not only useful in the case of bad intentions, but to reconstruct why something has changed the way it did.

It's duplicated with the `action` table.

Reply:
Yes, but the action table is often enough deleted by users since it is **way** too large,
sometimes taking up half of the entire DB storage.
Plus, the action log is supposed to be complete, and I have no idea how complicated the logic will become if we decide to intermingle action table entries into it.
It sounds like an architecturally stupid idea to de-duplicate it.